### ISSUE TITLE
Use as 'voting'-system?

### ISSUE THREAD

**Issue by EternalBlack** [bymayo/craft-points#4]
Is there any possibility to use this plugin as kind of a voting system for entries?
Something like `user A` posted and entry, and `user B` and `user C` can down- or upvote (once) on that entry resulting in `user A` receiving or losing points. I.e. any user can post an entry and all others (including himself) can up- or downvote on that entry taking influence on the points of the user who posted.
I would assume because it is event-driven this won't be possible out of the box but maybe I got that wrong.

**Comment by madebyshape** [comment]
Currently not at the moment. But I'll consider it for future releases, but it's not likely to be at the top of my list. 

You can currently create entries so if a particular user creates an entry then it will give them points.

**Comment by ChaseGiunta** [comment]
@EternalBlack you might want to check out https://github.com/dukt/craft-like or https://craftpl.us/plugins/upvote . Both of which could be used for what you want to accomplish.

**Comment by aladrach** [comment]
I'd love to see this enhancement as well!

**Comment by madebyshape** [comment]
Yeah it is a good idea! Have you two tried using the Upvote or Like plugin? Obviously this doesn't change the users points direct but it's a start.

It's likely that the upvote plugin (Have one in the works currently) will be seperate to 'points' but they will interact with both.

Because essentially points isn't an upvoting plugin.

I plan to add a few improvements to the plugin over the Christmas break!

**Comment by EternalBlack** [comment]
I'm sorry, but I currently am not able to test things out and tinker around. What I'm in need of though is something that enables a user to rate any users every-action (i.e. voting for a file a user posted or voting for a comment a user made).

But I'm really glad that you are planning on building/integrating something. Thanks in advance!
